Docs/Ratings/rating.html

```diff
@@ -0,0 +1,261 @@
+                        Alerts
+                    </a>
+                    <a href="../Badges/badge.html" class="tr-btn tr-btn-transparent tr-btn-icon-label">
+                        Badges
+                    </a>
+                    <a href="../buttons/buttons.html" class="tr-btn tr-btn-transparent tr-btn-icon-label">
+                        Buttons
+                    </a>
+                    <a href="../Cards/card.html" class="tr-btn tr-btn-transparent tr-btn-icon-label">
+                        Cards
+                    </a>
+
+                    <a href="../Headers/header.html" class="tr-btn tr-btn-icon-label tr-btn-transparent">
+                        Headers
+                    </a>
+
+                    <a href="../Images/image.html" class="tr-btn tr-btn-icon-label tr-btn-transparent">
+                        Image
+                    </a>
+                    <a href="../input/input.html" class="tr-btn tr-btn-icon-label tr-btn-transparent">
+                        Input
+                    </a>
+                    <a href="../Text-Utilities/text-utilities.html" class="tr-btn tr-btn-icon-label tr-btn-transparent">
+                        Text Utilities
+                    </a>
+                    <a href="../Lists/lists.html" class="tr-btn tr-btn-icon-label tr-btn-transparent">
+                        Lists
+                    </a>
+                    <a href="../Modals/modal.html" class="tr-btn tr-btn-icon-label tr-btn-transparent">
+                        Modals
+                    </a>
+                    <a href="#" class="tr-btn tr-btn-icon-label tr-btn-transparent">
+                        Ratings
+                    </a>
+                    <a href="../Snackbars/snackbar.html" class="tr-btn tr-btn-icon-label tr-btn-transparent">
+                        Snackbars
+                    </a>
+                    <a href="../Grid/grids.html" class="tr-btn tr-btn-icon-label tr-btn-transparent">
+                        Grid layouts
+                    </a>
+                </ul>
+                
+            </div>
+        </aside>
+    </div>
+
+    <header class="tr-header main-header d-flex f-wrap">
+        <button data-target="sidebarMain" class="tr-btn tr-btn-icon hamburger"><i class="fas fa-bars"></i></button>
+        <div class="tr-heading">
+            <a href="../../index.html">
+                <h2 class="header-brand">
+                    <img class="logo" src="../../assets/tarang-favicon-2.png" alt="Tarang Logo">
+                    <span>arang</span> <span>UI</span>
+                </h2>
+            </a>
+        </div>
+        <nav class="tr-nav d-flex">
+            <a href="#" class="tr-btn tr-btn-icon-label tr-btn-link">
+                Docs
+                <i class="fas fa-file"></i>
+            </a>
+            <a href="https://github.com/kjais1720/Tarang-UI" class="tr-btn tr-btn-icon-label tr-btn-link">
```

Comment:
add _blank attributes , page  open in new tab

Reply:
Thank you for pointing it out, I had added it everywhere but forgot to put it here